docs/_series/phockcyber.md

```diff
@@ -0,0 +1,38 @@
+---
+name: PHOCKCYBER
+author: KAWABANGGA_F
+image: https://i.imgur.com/oPzi4eQ.gif
+video: https://i.imgur.com/JpNB6Jx.mp4
+date: 2022-03-02
+description: (function repeat() {
```

Comment:
Description needs to be in quotes (either single or double) if you want to use a code. However, it will NOT look good on the website.

Reply:
ok mario, i have updated the description. thank you